data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+	description `Compared to other vessels in the galaxy, the Gypsum is much slower due to the Gegno's technology level, and has a harder time keeping up with faster asteroids. For those asteroids it can catch up to, the powerful excavator drill is able to grind materials out with ease.`
+
+ship Slate
+	sprite ship/slate
+	thumbnail thumbnail/slate
+	swizzle 0
+	attributes
+		category Transport
+		licenses
+			"Gegno Civilian"
+			"Vi Evocati"
+		cost 336000
+		mass 121
+		bunks 7
+		"cargo space" 36
+		drag 1.7
+		"engine capacity" 58
+		"fuel capacity" 400
+		"heat dissipation" 0.8
+		hull 825
+		"outfit space" 125
+		"required crew" 1
+		shields 0
+		"weapon capacity" 12
+		weapon
+			"blast radius" 18
+			"shield damage" 40
+			"hull damage" 132
+			"hit force" 210
+	outfits
+		"C3 Scrap Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering"
+		"RG15 Torch Thruster"
+	engine 0 40
+	leak leak 60 50
+	explode "small explosion" 5
+	explode "tiny explosion" 10
+	description "The Slate is the Gegno Vi's variation of the Shale. It appears they took the original design and added bulkier systems with more space, a prime characteristic of Vi vessels. Slightly larger than other shuttleships to suit the Vi's stature, it's able to carry more than a few Gegno cadets along with a small cargo load."
+
+ship Schist
+	sprite ship/schist
+	thumbnail thumbnail/schist
+	swizzle 0
+	attributes
+		category "Light Freighter"
+		cost 1932000
+		licenses
+			"Gegno Civilian"
+			"Vi Evocati"
+		mass 560
+		bunks 86
+		"cargo space" 122
+		drag 5.9
+		"engine capacity" 106
+		"fuel capacity" 500
+		"heat dissipation" 0.7
+		hull 12375
+		"outfit space" 210
+		"required crew" 36
+		shields 0
+		"weapon capacity" 38
+		weapon
+			"blast radius" 71
+			"shield damage" 310
+			"hull damage" 660
+			"hit force" 700
+	outfits
+		"C7 Brawl Core"
+		Hyperdrive
+		"Irate Cannon" 2
+		"R02 Battlezone Battery"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine 0 150
+	gun -14 -169 "Irate Cannon"
+	gun 14 -169 "Irate Cannon"
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 10
+	explode "medium explosion" 25
+	explode "small explosion" 25
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description "The Schist is the Gegno Vi's variation of the Gegno's old mid-range Dolomite. It has a decent cargo size, and enough bunks to carry a whole platoon of Gegno warriors. It is the primary vessel used to transport recruits to the front lines, and veterans back to home territories." 
+	description `Although a relatively light armament is an exception for the Vi's very aggressive and combat-oriented ship design philosophy, Schists still carry a pair of cannon mounts to be able to bring something, rather than nothing, to a fight.`
```

Comment:
```suggestion
	description `	Although a relatively light armament is an exception for the Vi's very aggressive and combat-oriented ship design philosophy, Schists still carry a pair of cannon mounts to be able to bring something, rather than nothing, to a fight.`
```